src/main/java/com/todo/Hiplanner/controller/MemberController.java

```diff
@@ -68,33 +58,33 @@ public String joinProc(Member member, RedirectView rv,String id){
     }
 
     @ResponseBody
-    @RequestMapping("/IdCheck")
+    @RequestMapping("/idCheck")
```

Comment:
RESTful 하게 디자인하는 가이드 중에서는 kebabCase를 사용하는 것을 권장해요!

```suggestion
    @RequestMapping("/id/check") 또는 @RequestMapping("/id-check")
```

은 어떨까요??
추가로, 여기서도 HTTP Method를 넣으면 좋을 것 같아요!